src/postings/recorder.rs

```diff
@@ -225,10 +256,12 @@ impl Recorder for TfAndPositionRecorder {
         buffer_lender: &mut BufferLender,
         serializer: &mut FieldSerializer<'_>,
         heap: &MemoryArena,
+        doc_id_map: Option<&DocIdMapping>,
     ) {
         let (buffer_u8, buffer_positions) = buffer_lender.lend_all();
         self.stack.read_to_end(heap, buffer_u8);
         let mut u32_it = VInt32Reader::new(&buffer_u8[..]);
+        let mut doc_id_and_positions = vec![];
```

Comment:
why was lending not possible here

Reply:
this is of type (u32, Vec<u32>), we can't lend the Vec<u32>, since it needs to be owned.

I should also note, that I couldn't find any recorder serialize methods in the perf profile.